lib/pf/config/pfqueue.pm

```diff
@@ -0,0 +1,55 @@
+package pf::config::pfqueue;
+
+=head1 NAME
+
+pf::config::pfqueue -
+
+=cut
+
+=head1 DESCRIPTION
+
+pf::config::pfqueue
+
+=cut
+
+use strict;
+use warnings;
+use pfconfig::cached_hash;
+
+BEGIN {
+    use Exporter ();
+    our ( @ISA, @EXPORT, @EXPORT_OK );
+    @ISA = qw(Exporter);
+    @EXPORT = qw(%ConfigPfQueue);
+}
+
+tie our %ConfigPfQueue, 'pfconfig::cached_hash', 'config::Pfqueue';
```

Comment:
Either the hash should be named ConfigPfqueue or the namespace be named config::PfQueue

From what I see, it seems the first one would be correct